src/containers/ColorContainer.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import Button from '../components/Button';
+import { ColorView } from '../components/ColorView';
+import useRandomValues from '../hooks/useRandomValues';
+
+const ButtonRapper = {
+  marginBottom: '20px',
+};
+const ColorContainer = () => {
+  const [randomValues, addBlock, deleteLastBlock] = useRandomValues();
+  return (
+    <div>
+      <div style={ButtonRapper}>
+        <Button label="Add block" onClick={() => addBlock()} />
+        <Button
+          disabled={randomValues?.length === 0}
+          label="Delete block"
+          onClick={() => deleteLastBlock()}
+        />
+      </div>
+
+      <div>
+        <TransitionGroup className="list">
+          {randomValues?.length === 0 ? (
```

Comment:
divide render by several components (container, view) or several render-functions, cause container shouldn't contain html-code in most cases